gameboard/game.scss

```diff
@@ -0,0 +1,197 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
+
+$base-font-color: #fff;
+$btn-color: #243c60;
+$bgc-color: rgba(0, 0, 0, 0.45);
+$btn-color-hover: #051e42;
+$base-font-family: "Lato", sans-serif;
+$bg-url: url("./bg.png");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: $base-font-color;
+  font-family: $base-font-family;
+}
+
+.wrapper {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  background-image: $bg-url;
+
+  &__main {
+    background-color: $bgc-color;
+    height: 100%;
+    width: 90%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: center;
+    position: relative;
+  }
+}
+
+.timer {
+  position: absolute;
+  top: 5%;
+  &__counter {
+    font-size: 48px;
+    font-weight: bold;
+  }
+}
+.gameBoard {
```

Comment:
Those styles are connected with the `game` page so maybe just `board` would be a good enough name for a class. WDYT?